Userland/Libraries/LibC/unistd.cpp

```diff
@@ -295,6 +295,12 @@ ssize_t write(int fd, const void* buf, size_t count)
     __RETURN_WITH_ERRNO(rc, rc, -1);
 }
 
+ssize_t fdcopy(int srcfd, int dstfd, size_t count)
```

Comment:
Your new syscall defines count as `ssize_t`, typo here? 

```suggestion
ssize_t fdcopy(int srcfd, int dstfd, ssize_t count)
```

Reply:
I think it would make more sense to make it a size_t in the syscall definition as well, since the syscall returns EINVAL if it's negative (and there's no point being able to represent negative values if they are all rejected).